TEAMS.yaml

```diff
@@ -21,12 +21,13 @@
 cockroachdb/docs:
   triage_column_id: 0 # TODO
 cockroachdb/sql-experience:
-  triage_column_id: 9056630
+  aliases: [cockroachdb/sql-syntax-prs, cockroachdb/sqlproxy-prs]
```

Comment:
we own sqlproxy now?

Reply:
heh no, but since it's highly dependent on our pgwire implementation it's more relevant to us than sql-queries

long-term it could make sense for our team to own it, but we don't right now